solr/core/src/test/org/apache/solr/cloud/AliasIntegrationTest.java

```diff
@@ -238,51 +239,52 @@ public void testModifyMetadataV2() throws Exception {
     checkFooAndBarMeta(aliasName, zkStateReader);
   }
 
-  public void testModifyMetadataV1() throws Exception {
+  @Test
+  public void testModifyPropertiesV1() throws Exception {
     // note we don't use TZ in this test, thus it's UTC
     final String aliasName = getTestName();
     ZkStateReader zkStateReader = createColectionsAndAlias(aliasName);
     final String baseUrl = cluster.getRandomJetty(random()).getBaseUrl().toString();
-    HttpGet get = new HttpGet(baseUrl + "/admin/collections?action=MODIFYALIAS" +
+    HttpGet get = new HttpGet(baseUrl + "/admin/collections?action=ALIASPROP" +
         "&wt=xml" +
         "&name=" + aliasName +
-        "&metadata.foo=baz" +
-        "&metadata.bar=bam");
+        "&properties.foo=baz" +
```

Comment:
I think we want the V1 API prefix for this to be "property" since each param occurrence is one property by itself.  This is consistent with various other V1 APIs (See CollectionApiMapping.java search for "property").  (My comment here applies to many spots in the diff, not just here of course).

Reply:
I tend to like dot notation to be nounish things. A set of things collectively know as "properties" with member element ".foo" rather than an adjective "property" describing ".foo"

          "properties" : {
            "foo": "baz",
            "bar": "bam"
            }
vs 

          "property" : {
            "foo": "baz",
            "bar": "bam"
          }

The v1 api flattening should match what we do for v2 I think?

Reply:
What you have for V2 (a JSON API) — “properties” is good. I’m directing my feedback expressly at V1 for “property”, which is a param prefix, not JSON. Again, see the existing APIs like core properties which already set this precedent.